service-bus/router/Cargo.toml

```diff
@@ -9,6 +9,7 @@ license = "GPL-3.0"
 edition = "2018"
 
 [dependencies]
+ya-compile-time-utils = "0.1"
```

Comment:
ya_sb_router is an example so this might be moved into `[dev-dep..s]`

Reply:
Fixed!